### ISSUE TITLE
modelZoo中的NASNet

### ISSUE THREAD

**Issue by pengtaomach** [deeplearning4j/deeplearning4j#7319]
#### Issue Description
在初始化NASNet网络模型时，出现
Cannot calculate activation types if no inputs have been set (use addInputs(String...))的错误
如下图：
构建模型方法：
![image](https://user-images.githubusercontent.com/38904475/54666557-028a4500-4b25-11e9-8c0e-0aa7dfe6c076.png)
错误信息如下：
![image](https://user-images.githubusercontent.com/38904475/54666155-fc479900-4b23-11e9-9c96-101f8a92cffa.png)

#### Version Information
* Deeplearning4j：deeplearning4j-zoo-1.0.0-beta3
* platform information ：win7
#### Contributing
跟踪源码发现在设置超参时，在设置InputType之前，获取了图中每个层和顶点的激活大小映射，造成出现上述错误
![image](https://user-images.githubusercontent.com/38904475/54666267-4466bb80-4b24-11e9-9e0f-90f049673b15.png)
![image](https://user-images.githubusercontent.com/38904475/54666309-65c7a780-4b24-11e9-992b-281a9599b9e0.png)
![image](https://user-images.githubusercontent.com/38904475/54666349-7f68ef00-4b24-11e9-988d-b151152bec06.png)
![image](https://user-images.githubusercontent.com/38904475/54666376-8f80ce80-4b24-11e9-96ae-4794169d99f2.png)
![image](https://user-images.githubusercontent.com/38904475/54666398-990a3680-4b24-11e9-885e-a9471ebe140f.png)

**Comment by basedrhys** [comment]
Also having this issue, any fixes?

**Comment by treo** [comment]
@basedrhys does this happen to you with beta6?

**Comment by basedrhys** [comment]
@treo Yes, it does, updated just recently. Using the DL4J API from within Intellij Community 2019.3

**Comment by treo** [comment]
@AlexDBlack looks like a proper bug. The Inputs should be set up not after constructing the full graph, but instead as pretty much the first thing, as there is a dependency on them being available when the blocks are constructed.

**Comment by agibsonccc** [comment]
Closing — references a removed/rewritten subsystem (DataVec, NLP/Word2Vec, Model Zoo, Spark, UI, or Arbiter). These have been fundamentally rewritten or removed in the upcoming release. Please open a new issue if you experience similar problems with the new version.
